File: src/main/resources/output/IFC4X3_python/Entities/IfcRoot/IfcRelationship/IfcRelConnects/IfcRelConnectsStructuralMember/IfcRelConnectsStructuralMember.py
class IfcRelConnectsStructuralMember(IfcRelConnects):
    def __init__(self):
        self.RelatingStructuralMember: IfcStructuralMember = None
        self.RelatedStructuralConnection: IfcStructuralConnection = None
        self.AppliedCondition: IfcBoundaryCondition = None
        self.AdditionalConditions: IfcStructuralConnectionCondition = None
        self.SupportedLength: IfcLengthMeasure = None
        self.ConditionCoordinateSystem: IfcAxis2Placement3D = None

    # === EXTENDED BY ===
    # IfcRelConnectsWithEccentricity

File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRoot/IfcRelationship/IfcRelConnects/IfcRelConnectsStructuralMember/IfcRelConnectsStructuralMember.cs
public class IfcRelConnectsStructuralMember : IfcRelConnects
{
    public IfcStructuralMember RelatingStructuralMember { get; set; }
    public IfcStructuralConnection RelatedStructuralConnection { get; set; }
    public IfcBoundaryCondition AppliedCondition { get; set; }
    public IfcStructuralConnectionCondition AdditionalConditions { get; set; }
    public IfcLengthMeasure SupportedLength { get; set; }
    public IfcAxis2Placement3D ConditionCoordinateSystem { get; set; }

    // === EXTENDED BY ===
    // IfcRelConnectsWithEccentricity
}
